crates/issue/src/lib.rs

```diff
@@ -300,7 +300,7 @@ impl<T: Config> Module<T> {
     fn _execute_issue(
         executor: T::AccountId,
         issue_id: H256,
-        tx_id: H256Le,
+        _: H256Le,
```

Comment:
I think we should just remove it.

Reply:
Yea that what why I made this PR as draft:

> This fixes the issue, but I want to remove the tx_id argument altogether